src/Raven.Server/Web/Authentication/TwoFactorAuthenticationHandler.cs

```diff
@@ -100,9 +92,9 @@ public async Task ValidateTotp()
         {
             var period = TimeSpan.FromMinutes(sessionDuration);
             
-            if (_auditLogger.IsInfoEnabled)
+            if (LoggingSource.AuditLog.IsInfoEnabled)
             {
-                _auditLogger.Info($"Connection from {HttpContext.Connection.RemoteIpAddress} with new certificate '{clientCert.Subject} ({clientCert.Thumbprint})' successfully authenticated with two factor auth for {period}. Has limits: {hasLimits}]");
+                LogAuditFor(nameof(TwoFactorAuthenticationHandler), $"successfully authenticated with two factor auth for {period}. Has limits: {hasLimits}");
```

Comment:
Instead of period we should rather log date it expires.

Reply:
done